libs/community/langchain_community/document_loaders/mongodb.py

```diff
@@ -66,7 +68,22 @@ async def aload(self) -> List[Document]:
                 "database": self.db_name,
                 "collection": self.collection_name,
             }
-            result.append(Document(page_content=str(doc), metadata=metadata))
+            if self.field_names is not None:
+                """Filter fields based on field_names"""
+                try:
+                    fields = {name: doc[name] for name in self.field_names}
+                except KeyError as err:
+                    logger.warning(f"{err.args[0]} field not found in Mongo document.")
+                    continue  """Skip this document if a specified field is not found"""
```

Comment:
`"""` is used for doc-strings for functions / classes, not for in code documentation

```suggestion
                    continue  # Skip this document if a specified field is not found
```

seme for changes below

Reply:
Fixed